apps/ledger-live-desktop/src/renderer/families/elrond/AccountSubHeader.tsx

```diff
@@ -29,42 +37,53 @@ const CustomButton = styled(Button)`
   border: none;
   padding-right: 14px;
 `;
-const AccountSubHeader = () => {
+const AccountSubHeader = ({ account }: AccountSubHeaderPropsType) => {
   const [isDrawerOpen, setIsDrawerOpen] = useState(false);
   const { t } = useTranslation();
+  const { elrondResources } = account;
+
   const openDrawer = () => {
     setIsDrawerOpen(true);
   };
   const closeDrawer = () => {
     setIsDrawerOpen(false);
   };
+
   return (
-    <Card px={2} py={1} mb={3}>
-      <CardContent>
-        <CardHeaderContainer>
-          <InfoCircle size={12} />
+    <Fragment>
+      <Card px={2} py={1} mb={3}>
+        <CardContent>
+          <CardHeaderContainer>
+            <InfoCircle size={12} />
+
+            <CardHeader>{t("elrond.account.subHeader.cardTitle")}</CardHeader>
+          </CardHeaderContainer>
 
-          <CardHeader>{t("elrond.account.subHeader.cardTitle")}</CardHeader>
-        </CardHeaderContainer>
+          <CustomButton outline={true} onClick={openDrawer}>
+            <Box horizontal={true} flow={1} alignItems="center">
+              <Box fontSize={3}>
+                <Trans i18nKey="elrond.account.subHeader.moreInfo" />
+              </Box>
 
-        <CustomButton outline={true} onClick={openDrawer}>
-          <Box horizontal={true} flow={1} alignItems="center">
-            <Box fontSize={3}>
-              <Trans i18nKey="elrond.account.subHeader.moreInfo" />
+              <ChevronRight size={12} />
             </Box>
+          </CustomButton>
+        </CardContent>
 
-            <ChevronRight size={12} />
-          </Box>
-        </CustomButton>
-      </CardContent>
+        <AccountSubHeaderDrawer
+          isOpen={isDrawerOpen}
+          closeDrawer={closeDrawer}
+          team="MultiversX"
+          family="MultiversX"
+        />
+      </Card>
 
-      <AccountSubHeaderDrawer
-        isOpen={isDrawerOpen}
-        closeDrawer={closeDrawer}
-        team="MultiversX"
-        family="MultiversX"
-      />
-    </Card>
+      {elrondResources.isGuarded && (
```

Comment:
You need the same condition on the LLM part, right?

Reply:
Theoretically yes, but unfortunately we don't have access to the `account` object in the component where we could place it as far up top in the page as desired, so we will show the warning regardless of the guardian status, thus the variable isn't used. I've already discussed it [here](https://discord.com/channels/885256081289379850/922464539193278544/1132960610989129768). 👍